Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/HxCalendarDoc/HxCalendar_TimeProvider.CodeSnippet.razor

```diff
@@ -0,0 +1,24 @@
+﻿<!-- Parameter -->
+<HxCalendar TimeProvider="usersTimeProvider" />
+
+<!-- Settings -->
+<HxCalendar Settings="UsersCalendarSettings" />
+
+<!-- Defaults -->
+<HxCalendar />
+@code {
+
+	static TimeProvider usersTimeProvider = new ZonedTimeProvider(TimeZoneInfo.FindSystemTimeZoneById("UTC-11"));
```

Comment:
Do not use `static` unless absolutely necessary (which is not this case I hope).